### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by eparis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,62 +7,39 @@ This repository manages the Cincinnati graph for OpenShift.
...
-
-```console
-$ ln -rs nodes/4.1/4.1.0.json channels/stable-4.1/
-```
-
-### Add an upgrade edge between releases
-
-Add JSON for an edge under the `edges` directory.
-The filename and subdirectory structure doesn't have a tooling impact, but the convention is to use `<to-major>.<to-minor>/<to-version>/<from-version>.json` to make it easy to get the history for separate z streams (e.g. `git log edges/4.1`).
+Edit the appropriete file in `channels/`. For example, to add a release to stable-4.2 you would edit `channels/stable-4.2.yaml`
```
Comment (wking) [inline README.md]: s/appropriete/appropriate/

File: README.md [hunk 2]
```diff
@@ -7,62 +7,39 @@ This repository manages the Cincinnati graph for OpenShift.
...
 
-Schema:
+The file contains a list of versions. Please keep the versions in order. And LEAVE COMMENTS if you skip a versions.
```
Comment (wking) [inline README.md]: s/a versions/a version/

File: README.md [hunk 3]
```diff
@@ -7,62 +7,39 @@ This repository manages the Cincinnati graph for OpenShift.
...
 
-* `channels` (required array of strings): Channel names in which the edge exists.
-    This can be a single single channel to allow for per-channel metadata such as phased rollout start times.
-    It can also be multiple channels if you do not need channel-specific metadata.
-* `from` (required string): The release `version` from which the edge originates.
-* `to` (required string): The release `version` where the edge terminates.
+### Block Edges
```
Comment (wking) [inline README.md]: nit: "Add a release to a channel" is a level-three header with sentence case, so `### Block edges` to match?  Or make the existing headers title case

File: README.md [hunk 4]
```diff
@@ -7,62 +7,39 @@ This repository manages the Cincinnati graph for OpenShift.
...
 
-For an example, see [`edges/4.1/4.1.2/4.1.0.json`](edges/4.1/4.1.2/4.1.0.json).
-
-To create entries for edges baked into a release image's `release-metadata`, use `extract-edges`.
-For example, create edges to 4.1.0 with:
+Create/edit an appropriete file in `blocked_edges/`. For example: to block all incoming edges to a release create a file such as:
```
Comment (wking) [inline README.md]: nit: s/appropriete/appropriate/

nit: I like to have each sentence on its own line so I don't have to worry about wrapping and get more useful `git blame` output.  So I'd like a newline before `For example`, but I can use word diffs and such with whatever source style if this is just me being obsessive ;).

File: README.md [hunk 5]
```diff
@@ -7,62 +7,39 @@ This repository manages the Cincinnati graph for OpenShift.
...
+```yaml
+$ cat blocked-edges/4.2.11.yaml
```
Comment (wking) [inline README.md]: nit: The `$ cat ...` means this is a ```console block, not a  ```yaml block.  You could also do something like:

    For example, block all incoming edges to 4.2.11 with a `blocked-edges/4.2.11.yaml` like:

    ```yaml
    to: 4.2.11
    from: .*
    ```

File: README.md [hunk 6]
```diff
@@ -7,62 +7,42 @@ This repository manages the Cincinnati graph for OpenShift.
...
-Schema:
+### Add Releases To Channels
 
-* `metadata` (optional object)
-    * `url` (optional string): Canonical URI for the release; generally an Errata URI.
-* `payload` (required string): The by-digest pullspec for the release [container image][image].
-* `version` (required string): The name of the release.
-    Currently [a Semantic Versioning 2.0.0][SemVer] string.
+Edit the appropriate file in `channels/`.
+For example, to add a release to stable-4.2 you would edit `channels/stable-4.2.yaml`
```
Comment (wking) [inline README.md]: nit: missing a trailing period.

File: README.md [hunk 7]
```diff
@@ -7,62 +7,42 @@ This repository manages the Cincinnati graph for OpenShift.
...
 
-For an example, see [`nodes/4.1/4.1.0.json`](nodes/4.1/4.1.0.json).
+The file contains a list of versions. Please keep the versions in order. And LEAVE COMMENTS if you skip a version.
```
Comment (wking) [inline README.md]: nit: multiple sentences on one line.

**Comment by eparis** [comment]
Ok, hopefully this works!

**Pull request merged**